packages/breadcrumbs/src/styled/StyledBreadcrumbItem.spec.tsx

```diff
@@ -6,19 +6,41 @@
  */
 
 import React from 'react';
-import { render } from 'garden-test-utils';
+import { render, renderRtl } from 'garden-test-utils';
+import { getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
 import { StyledBreadcrumbItem } from './StyledBreadcrumbItem';
 
 describe('StyledBreadcrumbItem', () => {
   it('renders default styling', () => {
     const { container } = render(<StyledBreadcrumbItem />);
 
-    expect(container.firstChild).toHaveClass('c-breadcrumb__item');
+    expect(container.firstChild).toHaveStyleRule('line-height', DEFAULT_THEME.lineHeights.md);
```

Comment:
This should retain a unit less value rather than using a pixel value found in the default theme.